src/_WarmUp/ArrayListAPIWarmup.java

```diff
@@ -0,0 +1,27 @@
+package _WarmUp;
+
+import java.util.ArrayList;
+
+public class ArrayListAPIWarmup {
+	public static void main(String[] args) {
+		System.out.println("CONTAINS:");
```

Comment:
I like how you gave each section a introduction? title? line! It helps the viewer know what they're looking at. 🥇

Reply:
:D